.github/workflows/ci-tests.yml

```
@@ -0,0 +1,28 @@
+name: CI Tests
+
+on: [push, pull_request]
+
+jobs:
+  ci-test:
```

```suggestion
  test-build:
```

Again, a more specific name